python/cudf/cudf/_fuzz_testing/utils.py

```diff
@@ -152,6 +172,16 @@ def get_avro_schema(df):
     return schema
 
 
+def get_orc_schema(df):
+    ordered_dict = OrderedDict()
+
+    for col_name, col_dtype in df.dtypes.items():
+        ordered_dict[col_name] = get_orc_dtype_info(col_dtype)
```

Comment:
Should be able to initialize the `OrderedDict` with a generator instead of initializing then modifying:
```suggestion
    ordered_dict = OrderedDict((col_name, col_dtype) for col_name, col_dtype in df.dtypes.items())
```